src/app/search/search-filters.ts

```diff
@@ -838,26 +854,14 @@ export function searchFilters(
         return item.dimInfo.tag !== undefined;
       },
       hasLight(item: DimItem) {
-        const lightBuckets = ["BUCKET_CHEST",
-          "BUCKET_LEGS",
-          "BUCKET_ARTIFACT",
-          "BUCKET_HEAVY_WEAPON",
-          "BUCKET_PRIMARY_WEAPON",
-          "BUCKET_CLASS_ITEMS",
-          "BUCKET_SPECIAL_WEAPON",
-          "BUCKET_HEAD",
-          "BUCKET_ARMS",
-          "BUCKET_GHOST",
-          3448274439,
-          3551918588,
-          14239492,
-          20886954,
-          1585787867,
-          1498876634,
-          2465295065,
-          953998645
-        ];
-        return item.primStat && item.bucket && _.contains(lightBuckets, item.bucket.id);
+        const statHashes = new Set([
+          1480404414, // D2 Attack
+          3897883278, // D1 & D2 Defense
+          368428387 // D1 Attack
+        ]);
```

Comment:
Nice! Mind moving this constant out of the function?

Reply:
fixed